### PR TITLE
pdpb: add new error BOOTSTRAPPED

### PR DESCRIPTION
**Pull request by overvenus**
PTAL @siddontang @disksing

### CODE REVIEW

**Comment by siddontang** [comment]
LGTM

File: proto/pdpb.proto [hunk 1]
```diff
@@ -59,6 +59,7 @@ enum ErrorType {
     UNKNOWN = 1;
     NOT_BOOTSTRAPPED = 2;
     STORE_TOMBSTONE = 3;
+    BOOTSTRAPPED = 4;
```
Comment (ngaut) [inline proto/pdpb.proto]: s/BOOTSTRAPPED/ALREADY_BOOTSTRAPPED/

**Comment by siddontang** [comment]
PTAL @ngaut

**Comment by disksing** [comment]
LGTM

**Pull request merged**